main.sh

```diff
@@ -62,5 +63,12 @@ if gh release view "${tag}" &>/dev/null; then
     # https://cli.github.com/manual/gh_release_delete
     gh release delete "${tag}" -y
 fi
+
+gh_options=""
+
+if [[ "$draft" == "1" ]]; then
```

Comment:
Could you change this option to accept `true` and `false` instead of `0` and `1`?

Reply:
The option now accepts `true` for draft mode and anything else is considered as standard release mode. There are no `--no-draft` or `--release` option that makes parsing `false` useful unfortunately.